android/package.json

```diff
@@ -15,13 +15,13 @@
 	],
 	"minSDKVersion": "16",
 	"vendorDependencies": {
-		"android sdk": ">=23 <=25.x",
-		"android build tools": ">=23 <=25.x",
-		"android platform tools": ">=17 <=25.x",
+		"android sdk": ">=25 <=26.x",
+		"android build tools": ">=25 <=26.x",
+		"android platform tools": ">=25 <=26.x",
 		"android tools": "<=26.x",
-		"android ndk": ">=r8e <=r9",
+		"android ndk": ">=r11c <=r15c",
 		"node": ">=4.0 <=7.x",
-		"java": "<=1.8.x"
+		"java": ">=1.8.x"
```

Comment:
That should be [here](https://github.com/appcelerator/titanium_mobile/blob/master/android/build/common.xml#L173) if you want to test `1.7`